oag/ja-jp/content/topics/releasenotes/oag/oag-release-notes.htm

```diff
@@ -304,29 +316,29 @@ <h1 class="relnotes">未翻訳のドキュメント</h1>
             <MadCap:topicToolbarProxy data-mc-skin="/Project/Skins/okta-toolbar-no-expand.flskn" style="mc-topic-toolbar-items: ;" />
                 </div -->
                                     <!-- *********************** Footer rows ********************************* -->
-                                    <div class="footer2" data-test="footer">
+                                    <div class="footer2">
                                         <div>
-                                            <p class="copyright" data-test="copyrightNotice">&copy;  <span class="mc-variable okta-variables.Year variable">2023</span> <span class="mc-variable okta-variables.CompanyName variable">Okta, Inc</span>. All Rights Reserved. それぞれの商標は、それぞれの商標所有者に帰属します。
-	</p>
+                                            <p class="copyright">© Copyright <span class="mc-variable okta-variables.Year variable">2022</span> <span class="mc-variable okta-variables.CompanyName variable">Okta, Inc</span> All Rights Reserved. それぞれの商標は、それぞれの商標所有者に帰属します。
```

Comment:
The year changed to 2022, but it was correct before (2023). Do you know why Eugene?

Reply:
I'll investigate. Release notes are copied from long-lived translation branches. We have to update each LL branch with the correct release notes.
I should update the receiving script to not override release notes in the main branch - that will be easier to maintain